platform/android/plugin/godot_plugin_jni.cpp

```diff
@@ -114,16 +114,19 @@ JNIEXPORT void JNICALL Java_org_godotengine_godot_plugin_GodotPlugin_nativeEmitS
 	String signal_name = jstring_to_string(j_signal_name, env);
 
 	int count = env->GetArrayLength(j_signal_params);
-	const Variant *args[count];
+	Variant args[VARIANT_ARG_MAX];
+
+	ERR_FAIL_COND_MSG(count > VARIANT_ARG_MAX, "Maximum argument count exceeded!")
+
+	for (int i = 0; i < MIN(count, VARIANT_ARG_MAX); i++) {
 
-	for (int i = 0; i < count; i++) {
 		jobject j_param = env->GetObjectArrayElement(j_signal_params, i);
-		Variant variant = _jobject_to_variant(env, j_param);
-		args[i] = &variant;
+		if (j_param)
+			args[i] = _jobject_to_variant(env, j_param);
```

Comment:
This condition should use brackets to follow our c++ coding rules (better add clean code even if it was copied from somewhere else).

Reply:
Fixed :)